xarray/coding/cftime_offsets.py

```diff
@@ -0,0 +1,765 @@
+"""Time offset classes for use with cftime.datetime objects"""
```

Comment:
Assuming that most of this file is copied/adapted from pandas, please add copy of the pandas copyright notice at the top.

Reply:
I added a copy of the license for pandas to the top of the module.  Is that what you meant?  Should we do the same in `cftimeindex.py`?

Reply:
> I added a copy of the license for pandas to the top of the module. Is that what you meant? 

Yes, this is what I had in mind. A comment would also be OK rather than the docstring.

> Should we do the same in cftimeindex.py?

Sure, this is probably a good idea